DigitalLearningSolutions.Web/Helpers/RegistrationEmailValidator.cs

```diff
@@ -35,6 +36,39 @@ ICentresDataService centresDataService
             ValidateEmailAddresses(true, model, modelState, userService, centresDataService);
         }
 
+        public static void ValidateEmailsForInternalAdminRegistration(
+            int userId,
+            InternalAdminInformationViewModel model,
+            ModelStateDictionary modelState,
+            IUserDataService userDataService,
+            ICentresService centresService
+        )
+        {
+            if (model.CentreSpecificEmail == null || !modelState.HasError(model.CentreSpecificEmail))
+            {
+                if (model.CentreSpecificEmail != null &&
+                    userDataService.CentreSpecificEmailIsInUseAtCentre(model.CentreSpecificEmail, model.Centre!.Value) &&
+                    model.CentreSpecificEmail != userDataService.GetCentreEmail(userId, model.Centre.Value))

```

Comment:
> I will address the need to make 2 database queries to check against existing centre emails in https://softwiretech.atlassian.net/browse/HEEDLS-1002

Do I have to do anything now, or just keep this in mind I should change this in case you merge before me?

Reply:
You do not have to do anything now